test/scenarios/jmx/docker-compose-kafka.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
+services:
+  kafka:
+    environment:
+      KAFKA_ADVERTISED_HOST_NAME: kafka
+      KAFKA_ADVERTISED_PORT: 9092
```

Comment:
Not sure why this file is needed. These properties are still deprecated in versions less than Kafka3 and can be replaced by `KAFKA_ADVERTISED_LISTENERS`

Reply:
Yes, But I wanted to allow people that run versions prior to 3.0.0 to be able to run with their config unchanged.

Reply:
@OneCricketeer ok, think i got it... pushed a commit  to fix this